cns/service/main.go

```diff
@@ -78,6 +78,8 @@ const (
 	// 720 * acn.FiveSeconds sec sleeps = 1Hr
 	maxRetryNodeRegister = 720
 	initCNSInitalDelay   = 10 * time.Second
+
+	envVarDisableCNIConflistGeneration = "CNS_DISABLE_CNI_CONFLIST_GENERATION"
```

Comment:
is making this opt-out instead of opt-in the safest way to roll it out?

Reply:
hm yeah I suppose we could make it opt-in, the real purpose of this is to just override the option in config so we can enable in some places and disable in others, so I think it's probably the same either way?

Reply:
actually no that's not true, if we want to only start to roll the feature when CNS supports this env var option, we should leave it disabled in config and then explicitly override in the env var. Will change